includes/download-functions.php

```diff
@@ -1018,6 +1018,40 @@ function edd_get_file_download_method() {
 }
 
 /**
+ * Returns if a user can access a download of a purchase
+ *
+ * @since 2.4
+ * @author Chris Christoff
+ * @param bool $bool Default for has_access
+ * @param mixed $purchase_data Array of purchase data 
+ * @param mixed $args Array of arguments
+ * @return nool If a user can access the file
+ */
+function edd_can_access_download( $bool = true, $purchase_data = array(), $args = array() ) {
```

Comment:
`$bool` should be `$has_access`, if someone stumbles upon the function, seeing `$bool` gives no context.